modules/microshift-adding-service-to-blueprint.adoc

```diff
@@ -23,6 +23,15 @@ groups = []
 name = "microshift"
 version = "4.12.0-1"
 
+[[packages]]
+name = "microshift-greenboot"
+version = "4.12.0-1"
```

Comment:
A nit: Greenboot will only be released in 4.13, so we may want to replace ALL versions in this example from 4.12.0-1 to 4.13.0-1.